Production_scripts/Process_extracts/extract_home_care.R

```diff
@@ -0,0 +1,139 @@
+#####################################################
+# Home Care Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input - Social Care Home Care Episodes
+# Description - Process Home Care Extract
+#####################################################
+
+
+# Load packages
+library(dplyr)
+library(dbplyr)
+library(createslf)
+
+
+year <- check_year_format("1920")
+
+
+# Read in data---------------------------------------
+
+source_hc_data <- readr::read_rds(get_sc_hc_episodes_path(update = "Jun_2022")) %>%
```

Comment:
```suggestion
source_hc_data <- readr::read_rds(get_sc_hc_episodes_path(update = latest_update())) %>%
```
Should this have the latest_update function here? We have just finished the latest update so i will update these functions to reflect this

Reply:
latest_update() has not been updated yet to be `Jun_2022` so costs are not read in